plugins/single_plugins/idle.cpp

```diff
@@ -58,6 +58,7 @@ class wayfire_idle
 
   public:
     bool idle_enabled = true;
```

Comment:
Why use idle_enabled when we have idle_ref ? Also, please choose something better than `idle_ref`, it isn't clear whether `idle_ref` is a count for things that want to enable or to disable idling.

Reply:
Since there is a toggle_idle function bound to a keybinding, to retain this functionality, it needs idle_enabled.